feat(config): add ThermoFloor z-temp2 thermostat

ivdimitro opened the pull request:
Added support for HeatIt Z-Temp2 battery operated thermostat

AlCalzone commented on packages/config/config/devices/0x019b/z-temp2.json:
```diff
@@ -0,0 +1,226 @@
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": {
+				"label": "Humidity reporting on change is disabled",
+				"value": 0
+			}
```
```suggestion
			"options": [
				"label": "Humidity reporting on change is disabled",
				"value": 0
			]
```

AlCalzone commented on packages/config/config/devices/0x019b/z-temp2.json:
```diff
@@ -0,0 +1,226 @@
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": {
+				"label": "Temperature reporting on change is disabled",
+				"value": 0
+			}
```
```suggestion
			"options": [
				"label": "Temperature reporting on change is disabled",
				"value": 0
			]
```

AlCalzone commented:
Did you write the file manually or import from OZW? Specifically the malformed options arguments?

AlCalzone commented:
Please disregard this comment. I'm trying to debug our bot
@zwave-js-bot fix lint

ivdimitro commented:
I first imported the config from OZW and then change the wording a bit and added the malformated "options". I fixed the "options" array but it failed in the index generation. I think this is not due to this configuration file in particular.

AlCalzone commented:
>  I think this is not due to this configuration file in particular.

It is, you mixed up min and max value on parameter #5 (or it is already wrong in the OZW file)

AlCalzone reviewed (changes requested):
see other comment

ivdimitro commented:
Finally all green. Sorry for the wasted time.
